ROADMAP.md

```diff
@@ -1,7 +1,45 @@
 # KF Serving 2019 Roadmap
-## Q2 2019
+## Q3 2019
+### v0.3 Performance and Stability (ETA: September 1, 2019)
+Objective: "Prevent feature regressions with 80% e2e test coverage against a live Cluster."
+* Automated End-to-End Tests
+    * Execute against a Kubeflow maintained GKE Cluster.
+    * Achieve >80% Test Coverage of Supported Features.
```

Comment:
Hard to achieve when the full feature list is not specified. Are you talking about code coverage of the code specific to this implementation?

Reply:
Agreed. 80% is a proxy for "good but not perfect" coverage. Features are often difficult to count.